nightwatch/conf/nightwatch.conf.js

```diff
@@ -49,27 +49,6 @@ const config = {
           "browser": 'Edge',
           "browser_version": '17.0',
         }
-      },
```

Comment:
Let's write up a ticket in this repo, or in the "design-docs" repo to test against mobile web browsers.